radixdlt-java-common/src/main/java/com/radixdlt/networks/Network.java

```diff
@@ -139,6 +139,10 @@ public int getId() {
     return id;
   }
 
+  public boolean sameAs(int id) {
```

Comment:
Is it really necessary?

Reply:
It enabled me to cleanup logic and reduce method complexity.

Reply:
Agree with @diegomrsantos . Creating another method to simplify and add another layer of indirection is not worth it:
`if (networkId == Network.MAINNET.getId()) {` to
`if (Network.MAINNET.sameAs(networkId)) {`

Reply:
The second version does not make the assumption that the reader knows about the meaning and properties of network ID (i.e. that all ID's must be distinct). Nevertheless, I see no point to argue and will roll back changes.

Reply:
The notion of equality is already defined in the equals method.